### PR TITLE
misc updates

### PR DESCRIPTION
**Pull request by abizer**
@kpengboy

### CODE REVIEW

File: _data/staff.yaml [hunk 1]
```diff
@@ -43,3 +43,9 @@
   title: Facilitator
   img: images/mcint.jpg
   description: I'm a 4th-year CS major who took some time off along the way. I like to talk about software philosophy, life, and living in the co-ops.
+
+- name: Kevin Peng
+  title: Facilitator
+  img: images/kpengboy.jpg
+  description: Kevin is a 5th-year EECS master's student, OCF Deputy Site Manager, and the only person who understands the OCF's constitution and by-laws.
+  style: "height: 200px;"
```
Comment (jvperrin) [inline _data/staff.yaml]: Why not just make the image 200x200 px in the first place so you don't have to do this? [Here's](https://www.gravatar.com/avatar/31e52c877dba9b4f7e5a0cd850af43f8?s=200&d=mm) a 200x200 version.

Comment (abizer) [reply _data/staff.yaml]: Thanks mine was definitely a hackjob

**Comment by abizer** [comment]
if they're all relative links that should be fine right? We can move to that in a future update.

**Comment by jvperrin** [comment]
Maybe? It has the link to the file in the instructions, and it provides a full link, so I don't see how we could make it relative really. Might be able to provide a sort of permalink essentially by putting it in the archive to begin with, so that it isn't broken later? Not sure that's a great solution though

**Comment by abizer** [comment]
what we might be able to do is spider the site and have like wget convert
all the links to relative and publish a tarball instead of the whole folder
which also causes a lot of spam in the git history.

File: _data/staff.yaml (hunk 1 above)
Comment (jvperrin) [reply _data/staff.yaml]: Don't forget to remove this line now, since it's not needed.

**Comment by abizer** [comment]
thanks

**Comment by jvperrin** [comment]
That sounds more complicated than necessary honestly. I mean having a spider or something to detect broken links makes sense as more of a one-off thing, but I don't feel like we'd want one for converting links to be relative. Maybe we can just provide the lab2 stuff via puppet and not have to worry about this in the first place? Just try and keep references like this out of the labs and only provide actual links (which we can then make relative)

**Comment by abizer** [comment]
I mean like when we move to archive this version of the site, instead of building it and moving `_site` to `archives/2017-fall/`, we'll just to something like `wget --mirror --convert-links --adjust-extension --page-requisites --no-parent -e robots=off https://decal.ocf.berkeley.edu && tar czvf fall-2017.tgz decal.ocf.berkeley.edu`

**Comment by kpengboy** [comment]
Why am I being listed as a facilitator? I always thought I was a guest lecturer...

**Comment by abizer** [comment]
Edit it if you please, I just wanted to make sure you were appropriately
credited.

File: _data/staff.yaml [hunk 2]
```diff
@@ -43,3 +43,9 @@
   title: Facilitator
   img: images/mcint.jpg
   description: I'm a 4th-year CS major who took some time off along the way. I like to talk about software philosophy, life, and living in the co-ops.
+
+- name: Kevin Peng
+  title: Facilitator
```
Comment (kpengboy) [inline _data/staff.yaml]: I would prefer "Guest Lecturer"

File: _data/staff.yaml [hunk 3]
```diff
@@ -43,3 +43,9 @@
...
+  img: images/kpengboy.jpg
+  description: Kevin is a 5th-year EECS master's student, OCF Deputy Site Manager, and the only person who understands the OCF's constitution and by-laws.
```
Comment (kpengboy) [inline _data/staff.yaml]: "Kevin is a 5th-year EECS master's student and OCF Deputy Manager." would be a sufficient description. I retired from DSM.

**Comment by kpengboy** [comment]
Thanks @abizer

**Pull request merged**